contents/blog/data-management-feature.md

```diff
@@ -0,0 +1,83 @@
+---
+date: 2022-03-23
+title: You can now organize your PostHog data
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+author: ["alex-kim"]
+categories: ["Product updates"]
+---
+
+As PostHog grows, the data that users are sending to PostHog continues to grow into the billions.
+
+Data explosion is real, and we've been working hard to deliver a solution to your data management woes. Today we're releasing our Data Management feature, which introduces new ways to help you organize events and properties being sent to PH. 
+
+This is the first step in a continuous effort to help make organizing your data easier:
+
+- Creating definitions for events and event properties
+- Managing data on the fly while building insights
+- New UI (Data Management tab and revamped definitions page)
+- Verifying trusted definitions
+- Categorizing your definitions with tags
+- Giving custom names and descriptions to events and properties
+- Making the right events and properties discoverable with event metrics
+
+## What is an Event?
+
+[Events](/docs/user-guides/events) are the basic building blocks of data that is sent to PostHog. Events can capture a wide variety of different interactions, from a new user signing up to capturing a server event when an API endpoint is called.
+
+[Event Properties](/docs/user-guides/events#event-properties) make up the meat of the event. These can be viewed nested in the context of a single event, or as a flat list in the Data Management > Event Properties tab.
+
+## How to create event and event property definitions
+
+### Creating definitions in the Insight Builder (on the fly!)
+
+Building an insight and see an event or event property that could be more descriptive? You can now create a definition for that without leaving the insight query builder.
+
+![PostHog - read definition hover card](../images/blog/data-management-feature/read_definition_card.png)
+
+![PostHog - write definition hover card](../images/blog/data-management-feature/write_definition_card.png)
+
+### Creating definitions in the new Data Management Tab
+
+Alternatively, click on view in the definition hover card and you'll be redirected to the new "Data Management" page. From here, explore a centralized place to manage all the event and event property definitions you have created.
+
+[WIP screenshot]
+
+### Creating definitions with the API
+
+You can also opt to create definitions with our API. Check out our documentation on how to use the [`event_definitions`](/docs/api/event-definitions) and [`property_definitions`](/docs/api/property-definitions) API endpoints.
+
+## How to use Data Management as a team (Best practices)
+
+1. **Rule of thumb**: The more definitions you define, the easier it will be for your teammates to understand what each event and property is describing. 
```

Comment:
```suggestion
1. **Rule of thumb**: The more definitions you add, the easier it will be for your teammates to understand what each event and property represents.
```